usaspending_api/references/tests/test_naics_autocomplete_v2.py

```diff
@@ -56,3 +57,23 @@ def test_naics_autocomplete_failure(client):
         content_type='application/json',
         data=json.dumps({'search_text': ''}))
     assert resp.status_code == status.HTTP_400_BAD_REQUEST
+
+@pytest.mark.django_db
```

Comment:
I recommend you create a new file to contain the tests. As long as the filename starts with `test` it will be added to the test runner

Reply:
sounds good, new file created as `test_load_naics.py`